From e2c076d8536ba79ceff7472522587b637cb400e0 Mon Sep 17 00:00:00 2001
From: Nikhil Devshatwar <a0132237@ti.com>
Date: Mon, 9 Dec 2019 13:56:29 +0530
Subject: [meta-arago][tisdk-setup-scripts][PATCH v1] setup-targetfs-nfs:
 Update command to replace Rules.make variables

Update the setup-targetfs-nfs.sh to replace Rules.make variables
so that they are initialized conditionally.

Signed-off-by: Nikhil Devshatwar <a0132237@ti.com>
---
 setup-targetfs-nfs.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/setup-targetfs-nfs.sh b/setup-targetfs-nfs.sh
index 60817ca..ac1611c 100644
--- a/setup-targetfs-nfs.sh
+++ b/setup-targetfs-nfs.sh
@@ -145,10 +145,10 @@ echo "EXEC_DIR or DESTDIR variable (depending on your SDK)."
 echo
 read -p "Press return to continue" REPLY
 
-sed -i "s=EXEC_DIR\=.*$=EXEC_DIR\=$dst/home/root/$platform=g" $cwd/../Rules.make
+sed -i "s=EXEC_DIR ?\=.*$=EXEC_DIR ?\=$dst/home/root/$platform=g" $cwd/../Rules.make
 check_status
 
-sed -i "s=DESTDIR\=.*$=DESTDIR\=$dst=g" $cwd/../Rules.make
+sed -i "s=DESTDIR ?\=.*$=DESTDIR ?\=$dst=g" $cwd/../Rules.make
 check_status
 
 echo "Rules.make edited successfully.."
-- 
2.17.1

